compiler/optimizer/StructuralAnalysis.cpp

```diff
@@ -58,7 +58,7 @@ void TR_RegionAnalysis::simpleIterator (TR_Stack<int32_t>& workStack,
       // The exit block must only be made part of the region headed by the entry
       // block. It provides an exit for all other regions.
       //
-      if (doThisCheck && next._succ.IsZero() &&
+      if (doThisCheck && next._succ.isEmpty() &&
```

Comment:
This comment isn't applicable to this line, but I need a place to put it.  :-)

Are these #includes still relevant in this file?  Please remove otherwise.
```
#include "cs2/arrayof.h"                              // for StaticArrayOf
#include "cs2/bitvectr.h"
#include "cs2/sparsrbit.h"
```